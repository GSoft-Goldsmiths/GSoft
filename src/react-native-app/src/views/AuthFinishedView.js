import React from 'react';

import FullPageView from '../components/FullPageView';

export default ({navigation}) => {
  const _buttonHandler = ()=>{
    navigation.navigate('AuthLoading');
  };
  return (
    <FullPageView
      imageSource={require('../../assets/images/icons/fireworks.png')}
      title="Congratulations"
      subtitle={"Everything is setting up now, let\'s start!"}
      buttonText="START"
      buttonHandler={_buttonHandler}
    />
  );
}