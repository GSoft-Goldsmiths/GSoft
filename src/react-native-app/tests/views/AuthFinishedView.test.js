import React from 'react';
import renderer from 'react-test-renderer';

import AuthFinishedView from '../../src/views/AuthFinishedView';

it('renders without crashing', () => {
  const rendered = renderer.create(<AuthFinishedView/>).toJSON();
  expect(rendered).toMatchSnapshot();
});
