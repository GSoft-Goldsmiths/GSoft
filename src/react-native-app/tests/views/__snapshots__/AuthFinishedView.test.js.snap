// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "paddingHorizontal": 20,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/images/icons/fireworks.png",
      }
    }
    style={
      Object {
        "height": 145,
        "marginBottom": 20,
        "width": 145,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#0093A8",
        "fontFamily": "Caviar Dreams",
        "fontSize": 30,
        "marginBottom": 15,
      }
    }
  >
    Congratulations
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#0093A8",
        "fontFamily": "Caviar Dreams",
        "fontSize": 15,
        "fontWeight": "500",
      }
    }
  >
    Everything is setting up now, let's start!
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0093A8",
        "height": 36,
        "justifyContent": "center",
        "marginVertical": 20,
        "width": 90,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "fontFamily": "Caviar Dreams Bold",
        }
      }
    >
      START
    </Text>
  </View>
</View>
`;
